crates/re_data_ui/src/annotation_context.rs

```diff
@@ -171,13 +176,23 @@ fn class_description_ui(ui: &mut egui::Ui, class: &re_components::ClassDescripti
                 .body(|mut body| {
                     re_ui::ReUi::setup_table_body(&mut body);
 
-                    for (from, to) in &class.keypoint_connections {
+                    // TODO(jleibs): Helper to do this with caching somewhere
```

Comment:
maybe also a profiling marker meanwhile, so we know when this is actually an issue :)